src/slice.rs

```diff
@@ -128,38 +128,52 @@ impl From<Vec<u8>> for SlicePayload {
     }
 }
 
+/// Helper function for [`create_random_slice_payload`].
```

Comment:
nit: all docstrings should start with a one-line paragraph
```suggestion
/// Helper function for [`create_random_slice_payload`].
///
```